tests/operator/operator.go

```diff
@@ -3384,3 +3362,29 @@ func patchKV(name string, data []byte) {
 		return err
 	}).WithTimeout(10 * time.Second).WithPolling(1 * time.Second).Should(Succeed())
 }
+
+var (
```

Comment:
couldn't this be an in-func `const`?

Reply:
> couldn't this be an in-func `const`?

Unfortunately, golang does not support constant objects, but only basic types.